x-pack/legacy/plugins/integrations_manager/server/packages/install.ts

```diff
@@ -45,20 +40,20 @@ export async function installPackage(options: {
 export async function installAssets(options: {
   savedObjectsClient: SavedObjectsClientContract;
   pkgkey: string;
-  asset: AssetType;
-  callCluster: CallESAsCurrentUser;
 }) {
-  const { savedObjectsClient, pkgkey, asset, callCluster } = options;
-  if (assetUsesObjects(asset)) {
-    const references = await installObjects({ savedObjectsClient, pkgkey, asset });
-    return references;
-  }
-  if (asset === 'ingest-pipeline') {
-    const references = await installPipelines({ callCluster, pkgkey });
-    return references;
-  }
+  const { savedObjectsClient, pkgkey } = options;
+
+  // Only install certain Kibana assets during package installation.
+  // All other asset types need special handling
+  const typesToInstall = [AssetType.visualization, AssetType.dashboard, AssetType.search];
```

Comment:
Are these 3 different than the other `SAVED_OBJECT_TYPES`? Could this be 
```suggestion
  const typesToInstall = Array.from(SAVED_OBJECT_TYPES);
```

Reply:
Yes, they are different. I only included those Kibana assets here of which I'm more or less sure, for now, that we can just copy them over.

Index patterns will need to be generated from `fields.yml` (we have tasks for that), and the other Kibana asset types are not listed in https://github.com/elastic/package-registry/blob/master/ASSETS.md so it's not specified what to do with them.